pkgs/development/libraries/libktorrent/default.nix

```diff
@@ -4,14 +4,14 @@
 }:
 
 let
-    mainVersion = "5.1";
+  mainVersion = "5.1.2";
```

Comment:
if ```mainVersion``` really is just ```version```, we should drop the mainVersion bit.

Reply:
I think `mainVersion` specifies ktorrent version while `version` specifies libktorrent version
In some cases `mainVersion` has format x.y while full ktorrent version has patch version, in other cases `mainVersion` has patch version as well and matches with ktorrent version. 

I'm not sure what should be done here. You could see different directory structure here used by ktorrent here https://download.kde.org/stable/ktorrent/

Reply:
That's messy. I agree, let's keep it.